app/tasks/build/dist.mjs

```diff
@@ -9,16 +6,16 @@ import { scripts, styles } from '../index.mjs'
 /**
  * Build review app task
  * Prepare dist folder for review app
+ *
+ * @type {import('govuk-frontend-tasks').TaskFunction}
  */
-export default gulp.series(
+export default (options) => gulp.series(
   task.name('clean', () =>
-    files.clean('**/*', {
-      destPath: join(paths.app, 'dist')
-    })
+    files.clean('*', options)
```

Comment:
By previously globbing `**/*` we were:

1. Generating more file system events for `gulp.watch()`
2. Causing race conditions by deleting children before parents

We only see 2) sometimes but happens more after package removal